servers_v6.json

```
@@ -90,5 +90,9 @@
   {
     "name": "Darkdustry",
     "address": ["mindurka.ru"]
+  },
+  {
+    "name": "Unjown",
```

```suggestion
    "name": "Unjown Server",
```

Ah yes rename Unjown to Unjown Server

---

sounds a bit redundant, its already a server and all the other groups also don't have that suffix 🤔